cypress/e2e/sign-in-out.cy.ts

```diff
@@ -0,0 +1,20 @@
+describe('Check sign out button (sign in is mocked)', () => {
+  it('make sure log out succeeds', () => {
+    // setup spying before request happens
+    cy.server();
```

Comment:
i think `.server()` is a deprecated function? I don't think you even need this line to run this function correctly

Reply:
oh, i thought i removed the other cy.server()'s when i was templating the intercept methods and then removed it after i found out it does nothing. thanks for catching this. 🙏